lib/iris/tests/unit/coords/test_Cell.py

```diff
@@ -85,10 +58,9 @@ def test_PartialDateTime_unbounded_cell(self):
     def test_datetime_unbounded_cell(self):
         # Check that cell comparison works with datetimes.
         dt = datetime.datetime(2000, 6, 15)
-        cell = Cell(datetime.datetime(2000, 1, 1))
-        # Note the absence of the inverse of these
-        # e.g. self.assertGreater(dt, cell).
-        # See http://bugs.python.org/issue8005
+        cell = Cell(cftime.datetime(2000, 1, 1))
```

Comment:
Changed this to `cftime.datetime` as I assume the usual way for a `datetime` object to appear in a cell is via `num2date`.  E.g.
https://github.com/SciTools/iris/blob/7db0bdb6acad80537ac33d3b051155dabfeb5e39/lib/iris/coords.py#L2180-L2183